openpype/settings/entities/schemas/system_schema/schema_modules.json

```diff
@@ -197,6 +197,20 @@
                 }
             ]
         },
+        {
```

Comment:
It is not module at all and it doesn't make sence to have option to turn it on/off as `enabled` is checked only in command line. I would remove it from settings...